codemeta.json

```diff
@@ -0,0 +1,52 @@
+{
+  "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+  "@type": "Code",
+  "name": "Berkeley Database Information Index (BDII)",
+  "description": "The BDII implementation of the information",
+  "provider": {
+    "@type": "Organization",
+    "name": "",
+    "url": ""
+  },
+  "maintainer": [
+    {
+      "@type": "Person",
+      "@id": "https://orcid.org/0000-0002-5686-3193",
+      "name": " Baptiste",
+      "familyName": "Grenier",
+      "affiliation": {
+        "@type": "Organization",
+        "name": "EGI Foundation",
+        "url": "http://www.egi.eu"
+      }
+    },
+    {
+      "@type": "Role",
+      "roleName": "Security and Vulnerability",
+      "url": ""
+    }
+  ],
+  "operatingSystem": [
+    "LINUX-2.4",
+    "centos 6",
+    "centos 7"
+  ],
+  "installUrl": "https://malandes.web.cern.ch/malandes/infosys/rpms/emi-bdii-site-1.0.2-1.el7.cern.noarch.rpm",
```

Comment:
I wanted to achieve parity with https://wiki.egi.eu/wiki/UMD_products_ID_cards so that there is a solid baseline from which to compare things.

I take the point - will open a  separate issue for this to be addressed in a subsequent release.

Reply:
addressed by #7 and #10